### ISSUE TITLE
Pre-trained model?

### ISSUE THREAD

**Issue by akskuchi** [Pendulibrium/ai-visual-storytelling-seq2seq#46]
Hello, 

Is there a way to download your pre-trained model for quickly inferring some samples?

Thanks!

**Comment by msmilevski** [comment]
A pre-trained model is not available.

All best,
Marko

**Comment by akskuchi** [comment]
Thanks for the update!

I was trying to understand if the image sequences in `6 Results and analysis` section of [the publication](https://arxiv.org/pdf/1805.05622.pdf) are from the test/validation data split?

**Comment by msmilevski** [comment]
Yes the image sequences are from the test set.
